string item0
string item1
string item2
---
#response
bool success
string message
